app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,110 @@
+@import 'colors';
+
+.vertical-separator {
+  margin: 0 25px;
+  max-height: 100%;
+  width: 3px;
+  border: 0;
+  background: linear-gradient(to top, $transparent, $orange, $transparent);
+  align-self: stretch;
+}
+
+.logo-section {
+  background-color: $white;
+  padding: 30px;
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.header-color-bar {
+  width: 100%;
+  height: 5px;
+  border-style: solid;
+  border-color: $orange;
+  background-color: $orange;
+}
+
+hr {
+  background: image-url('landing/triangle.png') repeat;
+  height: 10px;
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.features-section {
+  height: 100%;
+  background-color: $light-grey;
+  border: 0;
+  margin: 0;
+  padding: 30px;
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.feature {
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  font-family: 'Open Sans';
+  text-align: center;
+  color: $dark-grey;
+  padding: 5px;
+  width: 100%;
+  max-width: 250px;
+  min-height: 230px;
+}
+
+.feature-img {
+  max-height: 60px;
+  height: 100%;
+}
+
+.feature-p {
+  max-width: 200px;
+  color: $gray;
+}
+
+.footer {
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  background-color: $orange;
+  font-family: 'Open Sans';
```

Comment:
Since all text in the landing should have this font-family, try doing something like:

```scss
* {
  font-family: 'Open Sans', sans-serif; // Don't forget to add the secondary font family!
}
```